src/spacy/access.clj

```diff
@@ -0,0 +1,97 @@
+(ns spacy.access
+  (:require [buddy.sign.jwt :as jwt]
+            [schema.core :as schema]
+            [clojure.tools.logging :as log]
+            [yada.yada :as yada]
+            [yada.security]))
+
+;; TODO: should come from the environment
+(def jwt-key
+  (let [array (byte-array 64)]
+    (doto (java.security.SecureRandom.)
+      (.nextBytes array))
+    (String. array)))
+
+(def session "session")
+
+(defn- sign [claims]
+  (jwt/sign claims jwt-key))
+
+(defn- read-cookie [cookie]
+  (jwt/unsign cookie jwt-key))
+
+(defmethod yada.security/verify ::cookie
```

Comment:
Apparently the multi-method is deprecated? https://github.com/juxt/yada/blob/master/src/yada/security.clj#L18

I don't really understand why, and I don't know if that means we can't use it. I really like that implementation. 🙁